apps/estuary/lib/estuary/application.ex

```diff
@@ -4,10 +4,25 @@ defmodule Estuary.Application do
 
   def start(_type, _args) do
     [
-      Estuary.Quantum.Scheduler,
-      {Estuary.InitServer, []}
+      EstuaryWeb.Endpoint,
+      Estuary.Quantum.Scheduler
     ]
+    |> config_child()
     |> List.flatten()
     |> Supervisor.start_link(strategy: :one_for_one, name: Estuary.Supervisor)
   end
+
+  # Tell Phoenix to update the endpoint configuration
+  # whenever the application is updated.
+  def config_change(changed, _new, removed) do
+    EstuaryWeb.Endpoint.config_change(changed, removed)
+    :ok
+  end
+
+  defp config_child(child) do
```

Comment:
Is there a better name for this function?  I think I can see what it's doing, but `config_child` doesn't seem quite right... maybe just `init`?